nuttx/arch/arm/Kconfig

```diff
@@ -253,52 +78,25 @@ config ARCH_CORTEXM4
 	select ARCH_HAVE_HIPRI_INTERRUPT
 	select ARCH_HAVE_SYSRESET
 
-config ARCH_CORTEXA5
-	bool
-	default n
-	select ARCH_HAVE_IRQPRIO
-	select ARCH_HAVE_MMU
-	select ARCH_USE_MMU
-	select ARCH_HAVE_COHERENT_DCACHE if ELF
-
-config ARCH_CORTEXA8
-	bool
-	default n
-	select ARCH_HAVE_IRQPRIO
-	select ARCH_HAVE_MMU
-	select ARCH_USE_MMU
-	select ARCH_HAVE_COHERENT_DCACHE if ELF
-
 config ARCH_FAMILY
 	string
-	default "arm"		if ARCH_ARM7TDMI || ARCH_ARM926EJS || ARCH_ARM920T
-	default "armv6-m"	if ARCH_CORTEXM0
-	default "armv7-a"	if ARCH_CORTEXA5 || ARCH_CORTEXA8
```

Comment:
My thinking was to trim the fat of any variables or configurations not supported by the STM32L4 series.